docs/fundamentals/toc.yml

```
@@ -942,6 +942,28 @@ items:
           - name: Overview
             displayName: "language rules"
             href: code-analysis/style-rules/language-rules.md
+          - name: this and Me preferences
+            items:
+            - name: IDE0003 and IDE0009
+              href: code-analysis/style-rules/ide0003-ide0009.md
+          - name: Use language keywords instead of framework type names for type references
```

```suggestion
          - name: Use language keywords for types
```
It's good to keep TOC titles as short as possible to make it easier to browse.